src/Types/Ada.purs

```diff
@@ -0,0 +1,56 @@
+module Types.Ada
```

Comment:
If this is only for `ProtocolParametersAlonzo`, I would prefer to get rid of this module. We already have the `Coin` type and it will get confusing having both, I think. There are some useful functions at the bottom we can just move to `Types.Value` to use with `Coin`